Build failure on Ubuntu 20.04 with liberica-17

Repro: Build the project on Ubuntu 20.04, with liberica-17 and IDEA 2021.2.2.

Error: 

```
> Task :buildSrc:generateGrammarSource UP-TO-DATE
> Task :buildSrc:compileJava NO-SOURCE
> Task :buildSrc:compileGroovy UP-TO-DATE
> Task :buildSrc:processResources NO-SOURCE
> Task :buildSrc:classes UP-TO-DATE
> Task :buildSrc:jar UP-TO-DATE
> Task :buildSrc:assemble UP-TO-DATE
> Task :buildSrc:generateTestGrammarSource NO-SOURCE
> Task :buildSrc:compileTestJava NO-SOURCE
> Task :buildSrc:compileTestGroovy NO-SOURCE
> Task :buildSrc:processTestResources NO-SOURCE
> Task :buildSrc:testClasses UP-TO-DATE
> Task :buildSrc:test NO-SOURCE
> Task :buildSrc:check UP-TO-DATE
> Task :buildSrc:copyModuleInfo UP-TO-DATE
> Task :buildSrc:build UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :compileTestJava NO-SOURCE
> Task :processTestResources NO-SOURCE
> Task :testClasses UP-TO-DATE
> Task :test NO-SOURCE
> Task :pretty:compileJava FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':pretty:compileJava'.
> 错误: 不支持发行版本 17
```

Fix: Add this to the `java` block in `build.gradle.kts`:

```kotlin
toolchain {
  languageVersion.set(JavaLanguageVersion.of(17))
}
```

I don't know why this is the case though, and why others are not encountering this problem.

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -41,6 +41,9 @@ subprojects {
     if (hasProperty("release")) withJavadocJar()
     sourceCompatibility = JavaVersion.VERSION_17
     targetCompatibility = JavaVersion.VERSION_17
+    toolchain {
+      languageVersion.set(JavaLanguageVersion.of(17))
+    }
   }
 
   if (useJacoco) jacoco {

